src/cli/common.js

```diff
@@ -197,3 +202,19 @@ function loadPluginBudget(baseDir: string): Promise<Budget | null> {
   const budgetPath = pathJoin(baseDir, "config", "pluginBudgets.json");
   return loadJsonWithDefault(budgetPath, pluginBudgetParser, () => null);
 }
+
+/**
+ * Load the currency details from config, falling back on defaults
+ * if need be.
+ */
+export async function loadCurrencyDetails(
+  currencyDetailsPath: string
+): Promise<CurrencyDetails> {
+  const {name, suffix} = await loadJsonWithDefault(
+    currencyDetailsPath,
+    currencyConfigParser,
+    defaultCurrencyConfig
+  );
+
+  return {name, suffix};
```

Comment:
The unnecessary destructuring into `{name, suffix}` makes this code brittle--now someone adding a new field would need to make a boilerplate change to this function too. Just do `return await loadJsonWithDefault(...)`

Reply:
Agreed, done.